Fitter/scripts/EFTPlotter.py

```diff
@@ -130,6 +134,29 @@ def GetWCsNLLFromRoot(self,base_name_lst,wc,unique=False):
         return [graphwcs,graphnlls]
 
 
+    def CreateNewLimitTreefor2DScan(self,name_lst,wc_lst):
+        graphwcs, graphnlls = self.GetWCsNLLFromRoot(name_lst,wc_lst,unique=True)
+        newRootFile = ROOT.TFile("tmp.root","RECREATE")          #temporary root file with new limit tree
+        limit = ROOT.TTree("limit","limit")
+        wc0 = array('f',[0])
+        wc1 = array('f',[0])
+        deltanll = array('f',[0])
```

Comment:
Why not just use `np.array` or even just `list`, so we don't have to import `array` up top?

Reply:
Thank you Brent. I switched to using `np.array` instead of importing `array`.